src/cdk/listbox/listbox.spec.ts

```diff
@@ -75,9 +75,9 @@ describe('CdkOption and CdkListbox', () => {
 
       expect(listboxEl.getAttribute('tabindex')).toBe('-1');
       expect(optionEls[0].getAttribute('tabindex')).toBe('0');
-    });
+    }));
 
-    it('should respect user given tabindex for focusable elements', async () => {
+    it('should respect user given tabindex for focusable elements', waitForAsync(async () => {
```

Comment:
I added `waitForAsync` to all of these tests, because any errors during the test weren't being picked up by Jasmine. This threw me off for a while since I could see that an error was being thrown, but my test wasn't failing.